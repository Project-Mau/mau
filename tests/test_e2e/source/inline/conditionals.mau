:+true:
:-false:

A true statement is [@if:true:&true]("TRUE", "FALSE").

A false statement is [@if:false:&true]("TRUE", "FALSE").
