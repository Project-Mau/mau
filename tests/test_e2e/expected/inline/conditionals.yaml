data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'A true statement is '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: 'TRUE'
          kwargs: {}
          subtype: null
          tags: []
          type: sentence
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: .
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'A false statement is '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: 'FALSE'
          kwargs: {}
          subtype: null
          tags: []
          type: sentence
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: .
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

